packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/index.tsx

```diff
@@ -73,7 +81,10 @@ export const CustomTypeBuilder = () => {
                       </DropdownMenuItem>
                       <DropdownMenuItem
                         color="tomato"
-                        disabled={customTypeSM.tabs.length <= 1}
+                        disabled={
+                          customTypeSM.tabs.length <= 1 ||
+                          tabWithUid?.key === tab.key
```

Comment:
❓ #question

I'm not sure of the product impact of this. Are you sure you want to block also all custom type tab that contains a UID? It seems bad to do that. Is it not only for page type that you want to do that?

Reply:
didn't think about that, will narrow down the condition to prevent deleting section with uid only on reusable pages

Reply:
It's maybe a product question if you're not sure 🙂

Reply:
that was already discussed, the decision was : prevent main tab from being deleted, and it was said in the context of the main tab containing uid be default: https://prismic-team.slack.com/archives/C02L3FN3AJK/p1722852039737899